packages/server/src/DirectConnection.ts

```diff
@@ -32,7 +32,7 @@ export class DirectConnection implements DirectConnectionInterface {
 
     transaction(this.document)
 
-    await this.instance.storeDocumentHooks(this.document, {
+    await this.instance.onStoreDocument(this.document, {
```

Comment:
This means that directConnection saves will be debounced too, we can disable this by passing `true` as the final arg but I feel like this should be preferable if your debounce is for rate-limit reasons?

Is there a reason it wasn't part of the throttle previously?